Assets/Scripts/VFX/RenderSettings/TimeLightingSettings.cs

```diff
@@ -0,0 +1,31 @@
+﻿using UnityEngine;
+using System.Collections;
+
+[System.Serializable]
+public struct TimeLightingSettings 
+{
```

Comment:
Data is the asset saved off. Need to have both because creating scriptable objects creates an asset which isn't needed in all cases.